projects/asn1-proto/asn1_proto_converter.cc

```diff
@@ -0,0 +1,166 @@
+#include "asn1_proto_converter.h"
+
+namespace asn1_proto {
+
+/* Returns number of bytes needed to encode num
+ * Where num is a variable-length integer with
+ * no leading zeros
+ */
```

Comment:
I believe most of libfuzzer follows the dominant Google style of `//` and aligned to 78/80 columns.

This is discussed some at https://google.github.io/styleguide/cppguide.html#Comment_Style

We also want to make sure it's a proper sentance, by adding the necessary articles. 
```suggestion
// Returns the number of bytes needed to encode |num| into a variable-length
// two's complement integer with no leading zeros.
```

This is trying to reword the comment so that it's clear what it's encoding *into*.